qemu/tests/block_performance_test.py

```diff
@@ -0,0 +1,106 @@
+import logging
+import time
+import re
+
+from virttest import utils_disk
+from virttest import utils_misc
+
+
+def run(test, params, env):
+    """
+    Test the performance improvement with option:queue-size/num-queues
+
+    1) Boot guest with two disk that with different option set
+    2) Format the disk
+    3) Use dd to test disk performance
+    4) Compare the result of two disk
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    def get_disk_op_cmd(disk_op_cmd, disk_size):
+        """
+        Find the disk driver letter and format the disk, and get
+        disk driver letter,return disk_op_command
+        """
+        if os_type == "windows":
+            disk_op_cmd = utils_misc.set_winutils_letter(session, disk_op_cmd)
+            logging.info("Get windows disk index that to be formatted")
+            disk_id = utils_disk.get_windows_disks_index(session, disk_size)
+            if not utils_disk.update_windows_disk_attributes(session, disk_id):
+                test.error("Failed to enable data disk %s" % disk_id)
+            d_letter = utils_disk.configure_empty_windows_disk(session,
+                                                               disk_id[0],
+                                                               disk_size)[0]
+            output_path = d_letter + ":\\test.dat"
+        else:
+            disks = utils_disk.get_linux_disks(session)
+            for key, value in disks.items():
+                if value[1] == disk_size and value[2] == "disk":
+                    output_path = key
```

Comment:
Better to use `get_linux_drive_path` from `avocado-vt/virttest/utils_misc.py`, maybe simple.

Reply:
This may make our code longer. Three disks require three serial numbers, and then we need to obtain the corresponding serial numbers. What do you think about this? The method we currently use will make the cfg file shorter.Thanks!